examples/example_batch_execution.py

```diff
@@ -0,0 +1,104 @@
+                                                   lag=pd.Timedelta(hours=24),
+                                                   train_if=lambda x, _: len(x["ClockShift"].indexes["time"]) > 0 and
+                                                                         x["ClockShift"].indexes["time"][0].hour == 0)
+    RollingRMSE(window_size=1, window_size_unit="d")(
+        y_hat=regressor_svr_power_statistics, y=pipeline["load_power_statistics"],
+        callbacks=[LinePlotCallback('RMSE'), CSVCallback('RMSE')])
+    return pipeline
+
+
+if __name__ == "__main__":
+    # Read the data via pandas.
+    data = pd.read_csv("../data/getting_started_data.csv", parse_dates=["time"], infer_datetime_format=True,
+                       index_col="time")
+
+    # Split the data into train and test data.
+    train = data[:6000]
+    test = data[8700:]
+
+    # Create all modules which are used multiple times.
+    regressor_svr = SKLearnWrapper(module=SVR(), name="Regression")
+    power_scaler = SKLearnWrapper(module=StandardScaler(), name="scaler_power")
+
+    # Build a train pipeline. In this pipeline, each step processes all data at once.
+    train_pipeline = Pipeline(path="../results/train")
+
+    # Create preprocessing pipeline for the preprocessing steps
+    preprocessing_pipeline = create_preprocessing_pipeline(power_scaler)
+    preprocessing_pipeline = preprocessing_pipeline(scaler_power=train_pipeline["load_power_statistics"])
+
+    # Addd the regressors to the train pipeline
+    regressor_svr(ClockShift=preprocessing_pipeline["ClockShift"],
+                  ClockShift_1=preprocessing_pipeline["ClockShift_1"],
+                  target=train_pipeline["load_power_statistics"],
+                  callbacks=[LinePlotCallback('SVR')])
+
+    print("Start training")
+    train_pipeline.train(data)
+    print("Training finished")
+
+    # Create a second pipeline. Necessary, since this pipeline has additional steps in contrast to the train pipeline.
+    pipeline = Pipeline(path="../results")
+
+    # Get preprocessing pipeline
+    preprocessing_pipeline = create_preprocessing_pipeline(power_scaler)
+    preprocessing_pipeline = preprocessing_pipeline(scaler_power=pipeline["load_power_statistics"])
+
+    # Get the test pipeline, the arguments are the modules, from the training pipeline, which should be reused
+    test_pipeline = create_test_pipeline(regressor_svr)
+    test_pipeline(ClockShift=preprocessing_pipeline["ClockShift"],
+                  ClockShift_1=preprocessing_pipeline["ClockShift_1"],
+                  load_power_statistics=pipeline["load_power_statistics"],
+                  callbacks=[LinePlotCallback('Pipeline'), CSVCallback('Pipeline')])
+
+    # Now, the pipeline is complete so we can run it and explore the results
+    # Start the pipeline
+    print("Start testing")
+    result = pipeline.test(test)
+    print("Testing finished")
```

Comment:
I get the following warning:

```
/home/home/VSCodeProjects/pyWATTS/venv/lib/python3.8/site-packages/sklearn/utils/validation.py:993: DataConversionWarning: A column-vector y was passed when a 1d array was expected. Please change the shape of y to (n_samples, ), for example using ravel().
  y = column_or_1d(y, warn=True)
```

Reply:
Should be another issue. The reason is the input_to_sklearn in the SKlearnWrapper